### ISSUE TITLE
MultiGPU training not avalilable on Sentence Transformer

### ISSUE THREAD

**Issue by tkngoutham** [UKPLab/sentence-transformers#1429]
I am using albert as pretraining model and trying to do finetune on sentece transformer using fit function in SentenceTransformer class.

But only encode function has multi proc execution. I did not understand why does fit function not available to run in multi-gpu execution.

**Comment by nreimers** [comment]
There is currently a PR on this:
https://github.com/UKPLab/sentence-transformers/pull/1215

Multi GPU training is not so simple, as multiple processes must be started, data be shared between them and gradients communicated back (depending on the loss function)
